recipe/meta.yaml

```diff
@@ -74,7 +74,10 @@ outputs:
       - epics/html
       - epics/include
       # windows .dll are under bin
+      - epics/bin/*/*.lib    # [win]
       - epics/lib/*/*.lib    # [win]
+      - Library/lib/*.lib    # [win]
+      - Library/bin/*.dll    # [win]
```

Comment:
Huh, that's interesting. Was that one of the issues?

Reply:
The files were not in the package, so something was wrong with the copying

But I now note 1-2 other issues: we test with a glob (which could hit 0 files), and we look for .lib files in a bin\ dir (where they should not be)